src/ralph/ui/forms/addresses.py

```diff
@@ -388,6 +389,10 @@ def __init__(self, *args, **kwargs):
     widget = IPWithHostWidget()
 
     def compress(self, value):
+        if value:
+            hostname, ip_number = value
+            if not ip_number:
+                raise ValidationError(_('IP Address is required'))
```

Comment:
There is not need know about veriables naming.  _('IP Address is required') give you everything what you need to know. 

Are you thinking that two nested "if" statement are little low shitdifficult to understand?